examples/jira-scan-workflow/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,562 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: argo-hub.jira-sonar-template.0.0.1
+  annotations:
+    argo-hub/version: '0.0.1'
+    argo-hub/description: 'Workflow template for a full deployment using jira, sonar and a canary rollout'
+    argo-hub/categories: 'argo'
+    argo-hub/license: 'MIT'
+    argo-hub/owner_name: 'Matthew Chung'
+    argo-hub/owner_email: 'matthew.chung@codefresh.io'
+    argo-hub/owner_avatar: 'https://github.com/matthewchungcodefresh.png'
+    argo-hub/owner_url: 'https://github.com/matthewchungcodefresh'
+    argo-hub/icon_url: "https://github.com/codefresh-io/argo-hub/blob/main/examples/jira-scan-workflow/assets/icon.svg"
```

Comment:
need to be change to this "https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/examples/jira-scan-workflow/assets/icon.svg"